flow.md

```diff
@@ -0,0 +1,40 @@
+# Flow
```

Comment:
I don't think we need this, as we now have flows in the tasks

Reply:
better to remove